server/integration_tests/test_data/demo_fallback/query_4/chart_config.json

```diff
@@ -115,27 +115,27 @@
             "unit": "%"
           },
           "Percent_Person_BingeDrinking": {
-            "name": "Percentage of Population Who Binge Drink",
+            "name": "%  Population Who Binge Drink",
```

Comment:
Sorry: `%  Population Who Binge Drink` feels still odd.  Esp starting title with `%`

How about:
`Prevalence of Binge Drinking`

Reply:
Good idea. Updated